infra/base-images/base-runner/coverage_helper

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+
+
+def get_shared_libraries(binary_paths):
```

Comment:
The logic was borrowed from https://cs.chromium.org/chromium/src/tools/code_coverage/coverage.py?l=386